packages/flame_forge2d/lib/body_component.dart

```diff
@@ -13,18 +13,25 @@ import 'forge2d_game.dart';
 /// seen
 abstract class BodyComponent<T extends Forge2DGame> extends Component
     with HasGameRef<T>, HasPaint {
+  BodyComponent({
+    Paint? paint,
+    int? priority,
+    this.renderBody = true,
+  }) : super(priority: priority) {
+    this.paint = paint ?? (Paint()..color = defaultColor);
+  }
+
   static const defaultColor = Color.fromARGB(255, 255, 255, 255);
```

Comment:
I think we can skip this one and rely on the default color of `HasPaint`

Reply:
I think we should do this on another PR since this would be a breaking change. In addition, it is out of the scope of the PR.

Reply:
Sounds good, I was thinking of it since you did some changes to paint in this PR too

Reply:
Oopsy. Did I change something? I believe I only reordered the constructor to right after the class declaration.